src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/AllowedCharactersBitmap.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is there a reason why you didn't do a `git mv` here? Would be preferable to preserve history.